File: BackEnd/10_week_Algorithms/1.선형_알고리즘_기초/1A.js
// 두 수 비교하기

const readline = require('readline')

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
})

function getMax(a, b) {
  if (a > b) {
    return a
  } else {
    return b
  }
}

rl.question('', (input) => {
  const [a, b] = input.split(' ').map(Number)
  answer = getMax(a, b)

  console.log(answer)

  rl.close()
})

File: Algorithm/10_week_Algorithms/1.선형_알고리즘_기초/1A.py
# 문제1A 최대값 함수

# a, b 중 최대값을 반환하는 함수
def getMax(a, b):
    if a>b:
        return a
    else:
        return b

a, b = map(int, input().split())
print(getMax(a, b))
